tests/src/JIT/Regression/JitBlue/DevDiv_284785/DevDiv_284785.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
There is no *required* failure code, I would say there is a *standard* code, or at least a commonly used failure code. IMO, zero is too easily misinterpreted.